### PR TITLE
moyo card first commit

### PR DESCRIPTION
**Pull request by Denis-Sizemov**
- [DEMO LINK](https://Denis-Sizemov.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Denis-Sizemov.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,63 @@
...
+        <div class="card__price-value">
+          $2,199
+        </div>
+      </div>
+
+      <a
+        href="#"
+        class="card__button"
+        data-qa="hover"
+        >
```
Comment (alexandra-protyanova) [inline src/index.html:58]:
```suggestion
      <a
        href="#"
        class="card__button"
        data-qa="hover"
      >
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,63 @@
...
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">
+          Product code: 195434
+        </div>
+      </div>
+
+      <div class="card__rate">
```
Comment (alexandra-protyanova) [inline src/index.html:30]: use more semantic tags I see only divs

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  color: #060b35;
+}
+
+.card {
+  width: 200px;
```
Comment (alexandra-protyanova) [inline src/style.css:16]: don't set width to bem block

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (alexandra-protyanova) [reply src/style.css:16]: width is still here, need to fix

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,118 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
```
Comment (alexandra-protyanova) [inline src/style.css:10]: move to `html` selector

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
```
Comment (alexandra-protyanova) [inline src/style.css:57]: create separate file for each bem block

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
fix all comments

**Pull request closed without merging**